### PR TITLE
grasshopper: Add version 1.1.4

### PR DESCRIPTION
**Pull request by TobiTenno**

### CODE REVIEW

**Comment by TobiTenno** [comment]
what exactly does "package-fix-needed" mean? I couldn't find anything from the label description (as there isn't one)

File: grasshopper.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            "url": "https://grasshopper.codingkiwi.de/client/latest.json"
+        }
+    },
+    "homepage": "https://grasshopper.coding.kiwi",
+    "checkver": {
+        "url": "https://grasshopper.codingkiwi.de/client/latest.json",
+        "jp": "$.version"
+    },
+    "license": "Freeware",
+    "description": "The free and easy SSH client"
```
Comment (Ash258) [inline grasshopper.json:20]:
```suggestion
    "version": "1.1.4",
    "description": "SSH client",
    "homepage": "https://grasshopper.coding.kiwi",
    "license": {
        "identifier": "Freeware",
        "url": "https://grasshopper.coding.kiwi/faq#free"
    },
    "architecture": {
        "64bit": {
            "url": "https://grasshopper.coding.kiwi/download/1-1-4-grasshopper-Setup-1-1-4-exe#/dl.7z",
            "hash": "d8950908ea59a08823d9faf1ea7dcfa74bdfbfcc59bf33fbbfca3db12ba71aaf",
            "pre_install": [
                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
            ]
        }
    },
    "bin": "graSSHopper.exe",
    "shortcuts": [
        [
            "graSSHopper.exe",
            "graSSHopper"
        ]
    ],
    "checkver": {
        "url": "https://grasshopper.codingkiwi.de/client/latest.json",
        "jsonpath": "$.version"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://grasshopper.coding.kiwi/download/$dashVersion-grasshopper-Setup-$dashVersion-exe#/dl.7z"
            }
        }
    }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Never use electron "portable" applications in official buckets. https://github.com/lukesampson/scoop-extras/pull/2087#issuecomment-488344372

image

File: grasshopper.json (hunk 1 above)
Comment (TobiTenno) [reply grasshopper.json:20]: yeah, see, stuff like that should actually be documented, not just left in an issue text and assume that i'm gonna find it in one of several thousand issues, lol.

File: grasshopper.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.1.4",
+    "description": "The free and easy SSH client",
```
Comment (Ash258) [inline grasshopper.json:3]: `The free` is stated in license property

```suggestion
    "description": "SSH client",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: grasshopper.json (hunk 2 above)
Comment (TobiTenno) [reply grasshopper.json:3]: The description is from their website. I'm aware it's in the license.

Comment (Ash258) [reply grasshopper.json:3]: What does website have to do with description?

Having same information twice is unnecessary.

Comment (TobiTenno) [reply grasshopper.json:3]: To quote the reference page:
> A one line string containing a short description of the program. Don’t include the name of the program, if it’s the same as the app’s filename.

This is short, descriptive, and does not repeat the file's name.

**Comment by TobiTenno** [comment]
@Ash258 I've changed what you asked for

File: grasshopper.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        "64bit": {
+            "url": "https://grasshopper.coding.kiwi/download/1-1-4-grasshopper-Setup-1-1-4-exe#/dl.7z",
+            "hash": "d8950908ea59a08823d9faf1ea7dcfa74bdfbfcc59bf33fbbfca3db12ba71aaf",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "bin": "graSSHopper.exe",
```
Comment (Ash258) [inline grasshopper.json:19]: What are available parameters?

Comment (TobiTenno) [reply grasshopper.json:19]: None that I'm currently aware of. I can submit a new PR when some are added. I've been in conversation with the maintainer for a few for features that are desired.

Comment (Ash258) [reply grasshopper.json:19]: Then remove the bin property

Comment (TobiTenno) [reply grasshopper.json:19]: Sure, I don't understand why, but ok.

**Review by Ash258 — CHANGES_REQUESTED** [review]
... `git mv grasshopper.json bucket/grasshopper.json`

**Review by Ash258 — APPROVED** [review]

**Comment by TobiTenno** [comment]
could this get merged?

**Pull request merged**